# Issue #163: Save issue with Model

**Repository:** esimtest_pro  
**Status:** Closed  
**Created:** 2025-02-04  
**Updated:** 2025-05-01  
**Closed:** 2025-05-01  
**Author:** @Muralipandiyan  
**Assignees:** @Muralipandiyan  
**Labels:** `bug`  

[View on GitHub](https://github.com/Simtestlab/esimtest_pro/issues/163)

## Description

First, I loaded a new model on eSimTest and provided values for the test case to pass. I then saved the passed test case in the verification folder. After that, I changed the expected values so that the test case would fail and saved the failed test case with the same name. I closed eSimTest and reopened it with the failed test case model. However, when I ran the model, it still passed.

![Image](https://github.com/user-attachments/assets/000165e7-8dc6-44ea-8bc8-85a9d47d492e)

@aeroramesh @AgilanArulchelvam @RajavelRajendiran @sajimotrax 